hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -3,6 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name:               "spi_device",
+  one_line_desc:      "Serial peripheral interface supporting different device modes, suitable for bulk-load data into and out of the chip",
```

Comment:
```suggestion
  one_line_desc:      "Serial peripheral interface supporting different device modes, suitable for bulk-load of data into and out of the chip",
```

Reply:
Thanks @johngt for correcting this. I took this from the approved spec but it sounded wrong to me :-)